daphne/src/hpke.rs

```diff
@@ -155,6 +158,16 @@ pub struct HpkeConfig {
     pub public_key: HpkePublicKey,
 }
 
+#[cfg(any(test, feature = "test-utils"))]
```

Comment:
Not super familiar with deepsize crate, but I assume its derive macro should automatically provide a proper implementation of this?  why does this need manual derivation, where we might forget to update this with changes in the future?

Reply:
We can only derive when all the members of a struct implement the trait, `HpkePublicKey` is a foreign type so we can't implement this foreign trait on it, hence we need to implement by hand.